cypress/support/commands.ts

```diff
@@ -24,17 +21,18 @@ Cypress.Commands.add('loadPage', (pageName) => {
   });
 });
 
-Cypress.Commands.add('iframe', (selector: string, options?: IframeOptions) => {
-  const DEFAULT_OPTIONS = {
-    // Cypress default timeout
-    timeout: 15000,
-  };
-
-  const ops = { ...DEFAULT_OPTIONS, ...options };
+Cypress.Commands.add('getIframeBody', (selector: string) => {
+  return cy
+    .get(selector, { timeout: 15000 })
+    .its(`0.contentDocument.body`)
+    .should('not.be.empty')
+    .then(cy.wrap);
+});
```

Comment:
I think being explicit about which part of the iframe we are selecting is better for these commands. Makes things more clear.